FlorenceBML/BMLacq679/BMLacq679.xml

```diff
@@ -38,93 +38,97 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     </msIdentifier>
```

Comment:
you cannot see the images because the facs still has the wrong n (98) and the online version I had fixed was overwritten on merge by yours - I had not noticed you did not update the `facs`.
Now you will be able to see them again but only until this branch is merged, unless you correct the value
please replace n="144"
(`<idno facs="Laurenziana/BML_005/BML_005-" n="144">`)

Reply:
OK. Done.

Reply:
The images no. 99-144 are still not visible to me. Yet alone small grey images/items are on the bottom line. I think I have to wait for a couple of minutes or until tomorrow.